qa/tasks/cephfs/test_volume_client.py

```diff
@@ -686,8 +678,7 @@ def test_get_authorized_ids(self):
             volume_id=volume_id,
         )))
         # Check the list of authorized IDs for the volume.
-        expected_result = None
-        self.assertEqual(str(expected_result), auths)
+        assert auths == 'None'
```

Comment:
We prefer `self.assertFoo` because it produces friendlier results in the error output.

Reply:
I wasn't aware of that. I'll reverse that change.